src/elements/Label/Label.js

```diff
@@ -49,27 +46,29 @@ function Label(props) {
     className
   )
 
-  const _props = {
-    className: classes,
-    onClick: handleClick,
-    ...rest,
-  }
-
-  const _detail = detail || detailLink
-  const detailComponent = (detailLink || onClickDetail) && 'a' || detail && 'div'
+  const DetailComponent = (detailLink || onDetailClick) && 'a' || 'div'
```

Comment:
Not seeing where the extra parens can be added here.  Here is another way of writing this:

```js
let DetailComponent = 'div'
if (detailLink || onDetailClick) DetailComponent = 'a'
```